bundle-workflow/Jenkinsfile

```diff
@@ -57,14 +49,15 @@ pipeline {
                     post() {
                         always {
                             cleanWs disableDeferredWipeout: true, deleteDirs: true
-                            sh "rm -rf ~/.m2/"
                         }
                     }
                 }
                 stage('build-x86') {
                     agent {
-                        node {
-                            label 'Jenkins-Agent-al2-x64-m5xlarge'
+                        docker {
+                            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+                            image 'opensearchstaging/ci-runner:al2-x64-arm64-jdk14-node10.24.1-20210924'
```

Comment:
This naming schema is throwing me, this is a multi-arch image, can we just use noarch instead of x64/arm64?

Reply:
There are many other arch like ppcle64, Z, and else.

Reply:
Is there documentation on the convention or common usages that I could see?

Reply:
I only see this as a improve item for later.
Right now I think it is ok to start with.